homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -46,11 +46,13 @@ class BMWSensorEntityDescription(SensorEntityDescription):
 
 SENSOR_TYPES: dict[str, BMWSensorEntityDescription] = {
     # --- Generic ---
-    "charging_time_remaining": BMWSensorEntityDescription(
-        key="charging_time_remaining",
-        icon="mdi:update",
-        unit_metric=TIME_HOURS,
-        unit_imperial=TIME_HOURS,
+    "charging_end_time": BMWSensorEntityDescription(
+        key="charging_end_time",
```

Comment:
Just want to make sure, this returns a UTC datetime timestamp?

Reply:
It is a tz-aware datetime object (https://github.com/bimmerconnected/bimmer_connected/blob/master/bimmer_connected/vehicle_status.py#L452) and displayed the correct time when testing.